include/robots/gazebo_tank.h

```diff
@@ -44,15 +45,17 @@ class GazeboTank
     {
         BOB_ASSERT(left >= -1.f && left <= 1.f);
         BOB_ASSERT(right >= -1.f && right <= 1.f);
-          // Set the velocity in the x-component
-        gazebo::msgs::Set(&msg, ignition::math::Vector3d(left * m_MaximumSpeed.value(), right * m_MaximumSpeed.value(), 0));
-        // Send the message
+        
+	// Set the velocity in the x-component
+        gazebo::msgs::Set(&msg, ignition::math::Vector3d(left * m_MaximumSpeed.value(), right * m_MaximumSpeed.value()));
```

Comment:
I have no idea how gazebo works but why are we now setting a ``Vector2d`` message as ``Vector3d``?

Reply:
Ah yes, I missed that one and then didn't build test it :wink: